scripts/new-release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+current=$(grep -Eo "version: '(.*)'" meson.build | grep -Eo "[0-9]+\.[0-9]+.[0-9]+")
+major=$(cut -d '.' -f1 <<< "$current")
+minor=$(cut -d '.' -f2 <<< "$current")
+patch=$(cut -d '.' -f3 <<< "$current")
+
+case $1 in
+    "major")
+        next=$(echo $(($major + 1)).0.0)
+        ;;
+    "minor")
+        next=$(echo $major.$(($minor + 1)).0)
+        ;;
+    "patch")
+        next=$(echo $major.$minor.$(($patch + 1)))
+        ;;
+    *)
+        echo "Don't know what to do, exiting!"
+        exit 1
+    ;;
+esac
+
+sed -i "s/$current/$next/" meson.build
```

Comment:
Has the chance of modifying other stuff that is not required

should problably match

`s/version: '$current'/version: '$next'/`